### ISSUE TITLE
No web content for ESP-WROVER-IE

### ISSUE THREAD

**Issue by Denore** [thelastoutpostworkshop/gpio_viewer#122]
Hi 
I am new to ESP 
Is it possible to add ESP-WROVER-IE to the supported list Please?
Or can I do it myself?

Thanks

**Comment by thelastoutpostworkshop** [comment]
Do you have a link to the board and a link to the pinout of your board ?

**Comment by Denore** [comment]
I hope this is what you need, Thanks so much for your quick reply.

https://docs.espressif.com/projects/esp-idf/en/stable/esp32/hw-reference/esp32/get-started-devkitc.html

![esp32-devkitC-v4-pinout](https://github.com/thelastoutpostworkshop/gpio_viewer/assets/4669679/a552d766-e2b6-4789-aa49-57ff6e9dd592)

https://www.espressif.com/sites/default/files/documentation/esp32-wrover-e_esp32-wrover-ie_datasheet_en.pdf

**Comment by thelastoutpostworkshop** [comment]
The board corresponding to this pinout in web application is "ESP32 VROOM 32D (38 pins)"

**Comment by Denore** [comment]
Hi Thanks for that but I am not getting as far as the Web App. The Tablet connects to the broadcast webapp from the ESP but there is no content on the webpage, Just the Tab label GPIOViewer. I presumed I had an unsupported device and that was my issue. I have set it up with the original libraries that you used on the youtube clip and when that didn't work I updated to the latest version of all the Libraries. I am at a loss what I have done wrong, I even started from scratch on a different Laptop and I still get the same result. I have made a simple webserver to control an LED just to confirm I can get it working. Any suggestion would be greatly appreciated.
